src/zcbenchmarks.cpp

```diff
@@ -472,3 +474,159 @@ double benchmark_listunspent()
     auto unspent = listunspent(params, false);
     return timer_stop(tv_start);
 }
+
+extern double benchmark_create_sapling_spend()
```

Comment:
We seem to compile just fine w/o the `extern` keyword here (and below). Is there any reason to have this here?

Reply:
It's a copy and paste from the header file... good spot.. removing.